qa/packaging/build.gradle

```diff
@@ -36,3 +36,13 @@ tasks.named("test").configure { enabled = false }
 tasks.register('destructivePackagingTest') {
   dependsOn 'destructiveDistroTest'
 }
+
+tasks.named('resolveAllDependencies') {
```

Comment:
I'd even be ok with us resolving _none_ of these as they are only required for BWC jobs, and each of them only tests a single version so they aren't downloading a lot.